test/unit/CookiesStorageTests/cookieSave_test.js

```diff
@@ -0,0 +1,52 @@
+goog.module('measurementLibrary.storage.CookiesStorage.testing.cookieSave');
+goog.setTestOnly();
+
+const CookiesStorage = goog.require('measurementLibrary.storage.CookiesStorage');
+
+describe('The save method for cookiesStorage', () => {
```

Comment:
So the tests that are included now have the default prefix (ml_) I will add some tests for empty prefix and one more non-empty custom prefix.

Reply:
Right, I just meant that since we provide an option to override that, there should be at least one test with a custom prefix.  And testing +1 testing the behavior when it's set to empty is a good idea too!